app/src/split/bluetooth/Kconfig

```diff
@@ -1,11 +1,11 @@
-# Copyright (c) 2022 The ZMK Contributors
+# Copyright(c) 2022 The ZMK Contributors
```

Comment:
Superfluous whitespace change.

Reply:
This and the below whitespace change is because the app/Kconfig file has a whitespace there and I like consistency.

Reply:
Maybe my brain is rotten.. but isn't this *removing* the whitespace?

Reply:
No, you're right and I'm being silly. Fixed.